tf2_geometry_msgs/test/test_tf2_geometry_msgs.py

```diff
@@ -102,7 +102,6 @@ def test_transform(self):
         self.assertEqual(out.pose.pose.position.x, 0)
```

Comment:
While we are modifying this file, let's remove the `#/usr/bin/env python3` at the top (that won't work on Windows).